docs/control-panel/field-manager/edit-field.md

```diff
@@ -0,0 +1,38 @@
+<!--
+    This source file is part of the open source project
+    ExpressionEngine User Guide (https://github.com/ExpressionEngine/ExpressionEngine-User-Guide)
+
+    @link      https://expressionengine.com/
+    @copyright Copyright (c) 2003-2020, Packet Tide, LLC (https://packettide.com)
+    @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+-->
+
+# Create/Edit Field
+
+**Control Panel Location: `Developer > Fields > New/Edit Field`**
+
+This section of the control panel allows you to create and edit [Fields](/fieldtypes/overview.md) to be used in channel entries.
+
+New field can be added by clicking "New Field" button, or by picking "Clone to New Field" from the existing field's "Save" dropdown - in that case, the original field will serve as "template" and the changes would be applied to new field only.
+
+If you are viewing certain field group when clicking "New Field" button, the field will be assigned to that group. The existing fields can be assigned to multiple field groups by editing the Field Group.
+
+Fields have the following settings:
+
+- **Type** -- The type of field. [List of available fieldtypes](fieldtypes/overview.md).
+- **Name** -- The descriptive name for the field which is displayed in the publish form. Unlike the Field Name, the label does not need to be unique within the system, so you can use the same label on multiple fields in different field groups.
```

Comment:
This looks like it isnt right - referring to the label. Is this supposed to be "Field Label"?